homeassistant/components/onewire/const.py

```diff
@@ -37,6 +37,20 @@
 }
 DEVICE_SUPPORT_SYSBUS = ["10", "22", "28", "3B", "42"]
 
+DEVICE_SUPPORT_PRECISION_MAPPING = ["28"]
+
+OWSERVER_PRECISION_MAPPING = {
```

Comment:
I think this should be keyed against the device family or the model:
```suggestion
DS18B20_PRECISION_MAPPING = {
```
or
```suggestion
PRECISION_MAPPING_FAMILY_28 = {
```

Reply:
Done